### ISSUE TITLE
`string-ascii` and `string-utf8` not supported

### ISSUE THREAD

**Issue by hstove** [Zondax/ledger-blockstack#78]
Hey, I recently found that this Ledger app does not support signing the `string-ascii` or `string-utf8` Clarity types. This unfortunately blocks a wide range of function calls from being used with a Ledger.

The full list of Clarity types (and their prefix) can be found here: https://github.com/blockstack/stacks.js/blob/master/packages/transactions/src/clarity/clarityValue.ts#L25

**Comment by neithanmo** [comment]
Investigating a bit about this, I found that there is no description for *StringASCII* and *StringUtf8* clarity values.. the referenced documentation [here](https://github.com/blockstack/stacks.js/blob/master/packages/transactions/src/clarity/clarityValue.ts#L23) points to  [here](https://github.com/stacksgov/sips/blob/main/sips/sip-005/sip-005-blocks-and-transactions.md#clarity-value-representation), where clarity values encoding is defined and the current ledger-app uses this official definition of clarity values on its implementation. Because there is no mention or definition of StringUtf8/ASCII we did not implement and probably won't add any implementation for those strings as they are not part of what we consider the *"Official"* clarity values definitions. unless docs are outdated, in this case I want to ask @kyranjamie for this. Are those clarity values officially supported, if so, where are their encoding description?
